lib/administrate/field/base.rb

```diff
@@ -16,6 +16,14 @@ def self.searchable?
         false
       end
 
+      def self.field_type
+        to_s.split("::").last.underscore
+      end
+
+      def self.permitted_attribute(attr, _options = nil)
+        attr
+      end
```

Comment:
When there's two, what do you think about doing:

```ruby
class << self
  def field_type
    to_s.split("::").last.underscore
  end
end
```

Do you think that helps or hinders readability?

Reply:
I think that pattern is more difficult to read. I feel that only the indent gives a hint of those methods being special in some way, particularly if there's a long list and the `class << self` line is not on screen.

Reply:
Yeah, I'm happy with that!